prototypes/entity.lua

```diff
@@ -107,13 +207,111 @@ data:extend({
 		{
 			shadow =
 			{
-				red = {0.26, -0.6},
-				green = {0.36, -0.6}
+				red = {0.56, -0.6},
+				green = {0.26, -0.6}
 			},
 			wire =
 			{
-				red = {-0.16, -0.9},
-				green = {0.16, -0.9}
+				red = {0.16, -0.9},
+				green = {-0.16, -0.9}
+			}
+		},
+	},
+	-- small storehouse
+	{
+		type = "container",
+		name = "storehouse-basic-small",
+		icon = "__Warehousing__/graphics/icons/storehouse-basic-small.png",
+		icon_size = 32,
+		flags = {"placeable-neutral", "placeable-player", "player-creation"},
+		minable = {mining_time = 2, result = "storehouse-basic-small"},
+		max_health = 250,
+		corpse = "big-remnants",
+		dying_explosion = "medium-explosion",
+		open_sound = { filename = "__base__/sound/machine-open.ogg", volume = 0.85 },
+		close_sound = { filename = "__base__/sound/machine-close.ogg", volume = 0.75 },
+		vehicle_impact_sound = { filename = "__base__/sound/car-metal-impact.ogg", volume = 0.65 },
+		resistances =
+		{
+			{
+				type = "fire",
+				percent = 90
+			}
+		},
+		collision_box = {{-0.95, -0.95}, {0.95, 0.95}},
```

Comment:
The collision box is meant to be smaller than the selection box. See the `-basic` elements—their collision boxes are 0.3 units smaller than their selection boxes.

Reply:
For all of these they didn't seem to have any effect but I also always play with the squeak through mod so maybe that's why I didn't notice anything while testing it.  We can absolutely make them 0.3 smaller though.  If it does have an effect if not using the squeak through mod.

Reply:
Making the collision box smaller than the apparent size of the entity makes it less annoying to drive around them (you can come close to colliding visually without damaging anything), and makes it possible to walk between adjacent buildings.

The [Factorio wiki says](https://wiki.factorio.com/Prototype/Entity#collision_box) it's customary to leave 0.1 units around buildings; Warehousing just has a customary border of 0.3 instead. You'll definitely notice a difference without Squeak Through.